### PR TITLE
Feature/project popup

### PR DESCRIPTION
**Pull request by 0xMALVEE**
project popup for desktop and mobile.
Added dynamically load project list from javascript array.

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -15,3 +19,235 @@ MobileMenuBtn.addEventListener('click', () => {
...
+                <defs>
+                <clipPath id="clip0_48_833">
+                <rect width="18" height="20" fill="white" transform="translate(3 2)"/>
+                </clipPath>
+                <clipPath id="clip1_48_833">
+                <rect width="18" height="20" fill="white" transform="translate(3 2)"/>
+                </clipPath>
+                </defs>
+                </svg>
+            </button>
```
Comment (tresorsawasawa) [inline scripts/index.js:202]: - [ ] As required for the project, kindly make sure your popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=tx1lGltRUgBrT1rM-0) (e.g., `Background color`, `Blur effect on the background`, `colors`, `alignment`, `spacings(padding & margin), `fonts(size, weight,...) on mobile and desktop`, ...)

### Figma Desktop Design
Screenshot from 2023-03-07 17-24-12

### Your Desktop Design
Screenshot from 2023-03-07 17-32-31

| Figma Mobile Design |  Your Moible Design |
| ------------------------------- | --------------------------- |
| Screenshot from 2023-03-07 17-36-14 | Screenshot from 2023-03-07 17-34-55 |

File: scripts/index.js [hunk 2]
```diff
@@ -15,3 +19,235 @@ MobileMenuBtn.addEventListener('click', () => {
...
+const htmlToInsert = ProjectPopup(MultiPostStories.id, MultiPostStories.title, MultiPostStories.techUsed, MultiPostStories.LiveLink, MultiPostStories.CodeLink, MultiPostStories.descriptionDefault, MultiPostStories.descriptionPopup, MultiPostStories.imageLink);
+WorkHead.insertAdjacentHTML('afterend', htmlToInsert);
+
+const FirstProjectBtn = document.getElementById('first-project');
+
+FirstProjectBtn.addEventListener('click', () => {
+  const PopUp = document.getElementById(`popup-${77}`);
+  PopUp.style.display = 'block';
+});
+
```
Comment (tresorsawasawa) [inline scripts/index.js:248]: - [ ] `README.md` file: Kindly ensure that all the contributors of your project with their working social media links are present in the `Authors` section of your README to make it more professional. 

- [ ] Also remember to fill your `Show your support` and `Acknowledgments` sections with their appropriate content to make your documentation more professional

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Team :wave: :wave:
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾🍾🎉**
Hello @0xMALVEE 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.
- Your linters are passing all the checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**